src/assets/icons/FacebookIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
+import Svg, { SvgProps, Path } from "react-native-svg"
+
+const FacebookIcon = (props: SvgProps) => (
+  <Svg
+    data-name="Component 1"
+    xmlns="http://www.w3.org/2000/svg"
+    width={17}
```

Comment:
Noticed the icons have different width and height values. Could this be the reason why they seem to have different sizes on the UI?

Reply:
From the design spec([link](https://xd.adobe.com/view/dd5d0245-b92b-4678-9d4a-48b3a6f48191-880e/screen/f5f3abf2-9384-460a-bac7-544c49648c0c/specs/)), there's also different sizes but we could try using 18 by 18 for width and heigh.

Reply:
I could just pass the height and width as a prop to the Icon component.